### PR TITLE
chore(docs): embedded typegate (v0.3.x) blog

### PR DESCRIPTION
**Pull request by destifo**
# Add a blog about Embedded Typegate.

[MET-564](https://linear.app/metatypedev/issue/MET-564/docs-embedded-typegate-v03x)

#### Migration notes

_No Migrations Needed_

- [ ] The change comes with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

## Summary by CodeRabbit

- **New Features**
  - Introduced a new blog post on emulating server nodes locally using the Embedded Typegate feature in Meta CLI.
  - Added a new `BlogIntro` component to the website for displaying styled blog introductions.

- **Documentation**
  - Updated documentation to explain how to spin up a local instance of Typegate for testing and development.

- **Chores**
  - Updated platform compatibility from "x86_64-linux" to "aarch64-darwin".

### CODE REVIEW

**Comment by destifo** [comment]
@coderabbitai review

File: website/blog/2023-03-15-emulating-servers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import BlogIntro from "@site/src/components/BlogIntro";
+
+
+# Emulating Your Server Nodes Locally
```
Comment (zifeo) [inline website/blog/2023-03-15-emulating-servers/index.mdx:4]: We capitalized only the first letter in the others' article titles and subtitles, let's remains consistent or change the others

**Review by zifeo — COMMENTED** [review]
Nice!

**Review by Yohe-Am — DISMISSED** [review]

File: website/shared/upgrade/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## Upgrade your Metatype development environment
+
+### Meta CLI
+<UpgradeMetaClI />
+
+### Typescript SDK
+<UpgardeTsSDK />
+
+### Python SDK
+<UpgradePythonSDK />
```
Comment (afmika) [inline website/shared/upgrade/index.mdx:17]: How about using tabs? Also since it will be used anywhere maybe we should remove any titles?

**Review by afmika — DISMISSED** [review]

**Review by Yohe-Am — APPROVED** [review]

**Review by Yohe-Am — APPROVED** [review]

**Pull request merged**